genshi/tests/input.py

```diff
@@ -278,6 +278,21 @@ def test_multibyte_character_on_chunk_boundary(self):
         self.assertEqual(1, len(events))
         self.assertEqual((Stream.TEXT, text), events[0][:2])
 
+    def test_convert_ElmentTree_to_markup_stream(self):
+        tree = et.fromstring(
+            u'<div class="test_div">text<span>some more text</span></div>'
+            )
+        ret = list(ET(tree))
+        self.assertEqual(repr(ret[0][1]), repr(
+            (QName('div'), 
+            Attrs([(QName('class'), 'test_div')])
+            )
+            ))
+        self.assertEqual(ret[1][0], 'TEXT')
+        self.assertEqual(ret[2][0], u'START')
+        self.assertEqual(ret[3][1], 'some more text')
+        self.assertEqual(ret[4][0], u'END')
+        self.assertEqual(repr(ret[5][1]), repr(QName('div')))
```

Comment:
Could we use the named constants here (e.g. `Stream.START` instead of `u'START'`) and make the asserts more similar in structure to the other tests?

E.g. `self.assertEqual((Stream.TEXT, "text"), events[1][:2])`.